vllm/distributed/parallel_state.py

```diff
@@ -98,11 +99,15 @@ def init_distributed_environment(
             init_method=distributed_init_method,
             world_size=world_size,
             rank=rank)
-        global _DEVICE_WORLD_GROUP, _CPU_WORLD_GROUP
+        global _DEVICE_WORLD_GROUP, _CPU_WORLD_GROUP, _SHM_WORLD_BUFFER
         _DEVICE_WORLD_GROUP = torch.distributed.group.WORLD
         ranks = list(range(torch.distributed.get_world_size()))
         _CPU_WORLD_GROUP = torch.distributed.new_group(ranks=ranks,
                                                        backend="gloo")
+        from vllm.distributed.device_communicators.shm_broadcast import ShmRingBuffer
+        _SHM_WORLD_BUFFER = ShmRingBuffer(pg=_CPU_WORLD_GROUP,
+                                          max_chunk_bytes=1024 * 1024,
```

Comment:
I'd do at least a megabyte, perhaps more

Reply:
I don't get it. I use `1024 * 1024`, which is 1MB.

Reply:
yep, correct, must have been half-asleep when I looked at this 😂